packages/desktop/src/cfg.rs

```diff
@@ -65,9 +66,16 @@ impl Config {
             root_name: "main".to_string(),
             background_color: None,
             last_window_close_behaviour: WindowCloseBehaviour::LastWindowExitsApp,
+            disable_default_menu_bar: false,
         }
     }
 
+    /// Set whether the default menu bar should be disabled.
+    pub fn with_disable_default_menu_bar(mut self, disable: bool) -> Self {
```

Comment:
I'm not sure about this name. It could leads to double negatives which can be confusing: `with_disable_default_menu_bar(false)` means the default menu bar is enabled. This is already a pattern that we have introduced into the desktop config builder, so I completely understand why you copied it, but I think `with_default_menu_bar` with a comment about the default being true might be more clear

Reply:
I agree, just tried to be consistent but actually found the naming kind of bad too in this case, will definitely adjust that to what you've suggested. I will probably make the changes later today.